pango/src/auto/context.rs

```diff
@@ -84,7 +84,7 @@ impl Context {
     #[doc(alias = "pango_context_get_language")]
     #[doc(alias = "get_language")]
     pub fn language(&self) -> Option<Language> {
-        unsafe { from_glib_full(ffi::pango_context_get_language(self.to_glib_none().0)) }
+        unsafe { from_glib_none(ffi::pango_context_get_language(self.to_glib_none().0)) }
     }
 
     #[doc(alias = "pango_context_get_matrix")]
```

Comment:
Further below, `fn metrics()` should return no `Option` according to your pango MR

Reply:
Indeed, fixed.